protostar/cheatable_starknet/callable_hint_locals/callable_hint_local.py

```diff
@@ -38,11 +35,11 @@ def _build(self) -> Callable:
         ...
 
     def build(self):
-        def wrapper(*args: Any, **kwargs: Any):
+        def wrapper(*args: Any, **kwargs: Any) -> ExecutionResult:
             try:
                 result = self._build()(*args, **kwargs)
                 return ValidExecution(ok=result)
-            except TransactionRevertException as ex:
-                return InvalidExecution(err_code=encode_shortstring(ex.message[:31]))
+            except BaseException as ex:
+                return InvalidExecution(err_code=encode_shortstring(str(ex)[:31]))
```

Comment:
I don't think we should be changing it. Some errors are non-recoverable in the test and I think tests should just outright fail in the general case.

Handling only `TransactionRevertException` here means returning an handleable error is an opt-in.

Reply:
Yes, I know, I probably should've noted that I'm about to fix this